### PR TITLE
Phase 0: Added reducers, interfaces, dummy data, and store for Quiz Features

### PR DESCRIPTION
**Pull request by omkarubale**

### CODE REVIEW

File: src/Kanbas/Database/quizzes.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[
```
Comment (bharatchawla53) [inline src/Kanbas/Database/quizzes.json:1]: Missing description, assignmentGroupId, quizTypeId, and accessCode.

File: src/Kanbas/store/interfaces/quizzes.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+export interface IKanbasQuizDetails extends IKanbasQuiz {
+  isShuffleAnswers: boolean;
+  timeLimit: number; // minutes
+  isMultipleAttempts: boolean;
+  showCorrectAnswersDate: Date;
+  isOneQuestionAtATime: boolean;
+  isWebcamRequired: boolean;
+  isLockQuestionsAfterAnswering: boolean;
```
Comment (bharatchawla53) [inline src/Kanbas/store/interfaces/quizzes.ts:24]: Do we still need a boolean for isNotifyUsersAboutChange? In the notes, we * to ask professor, did you ask him in his office hours?

Comment (omkarubale) [reply src/Kanbas/store/interfaces/quizzes.ts:24]: I forgot I had to ask that. We can keep it for now, and come back to it if needed.

File: src/Kanbas/Database/quizzes.json (hunk 1 above)
Comment (omkarubale) [reply src/Kanbas/Database/quizzes.json:1]: I used enums for assignmentGroup and quizType. AccessCode and Description added.

**Review by bharatchawla53 — APPROVED** [review]

**Pull request merged**